glue/app/qt/application.py

```diff
@@ -464,7 +465,7 @@ def _create_menu(self):
             for a in self._actions['session_export']:
                 submenu.addAction(a)
         menu.addSeparator()
-        menu.addAction("Edit &Settings", self._edit_settings)
+        # menu.addAction("Edit &Settings", self._edit_settings)
```

Comment:
Oops, this was accidental, fixed in https://github.com/glue-viz/glue/commit/f240960e27d9cb622abc086da58c061f01930cad